jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/forumAdministration.js

```diff
@@ -558,13 +565,22 @@ function sendForumConfiguration(e) {
 };
 
 /**
- * Show dialog for create group.
- * 
+ * Show dialog for create or edit group.
+ *
  * @param event
  */
-function showGroupCreateDialog(event) {
+function showGroupManagementDialog(event) {
     event.preventDefault();
-
+    // Create a new group or edit an existing?
+    var editMode = $(this).attr('id') == "editGroup";
+
+    if (editMode){
+        // find row with group and extract all data that we need.
+        var groupRow = $(this).parents('.group-edit-row');
+        var groupId = groupRow.attr('id').substring('group-id'.length);
```

Comment:
I'd rather create a hidden field or another attribute to eliminate additional logic with strings.